src/docker-publish/orb.yml

```diff
@@ -82,6 +82,46 @@ examples:
                       name: Do this after the build.
                       command: echo "Did this after the build"
 
+  build_without_publishing_job:
+    description: >
+      Build, but don't publish, an image using the publish job.
+
+    usage:
+      version: 2.1
+
+      orbs:
+        docker-publish: circleci/docker-publish@1.0.0
```

Comment:
Do these render into the docs? One thing I noticed is that the docs all have old (or just dummy versions) and so when I (or a general user) copies the examples, you get an error of not finding the orb on first build. Could the example version be derived from the same (most recent) in the header, for example here --> https://circleci.com/orbs/registry/orb/circleci/docker-publish

Reply:
@vsoch the examples are not meant to be copy-paste-able per-se... we've sort of defaulted to using `@1.0.0` as the version for any orb in an example, because it's simple & easy to understand, even though many orbs haven't even reached that semantic version yet.

Reply:
@vsoch updating the example to use the most recent published version is certainly possible, but given the number of orbs & the frequency of updates, i think it would be a bit of a headache...

Reply:
Haha, yeah I completely agree :)